spec/views/idv/in_person/ready_to_verify/show.html.erb_spec.rb

```diff
@@ -103,29 +103,36 @@
     end
   end
 
-  context 'USPS outage message flag' do
+  context 'Outage message flag' do
     before(:each) do
     end
 
-    it 'renders the USPS outage alert when flag is enabled' do
-      allow(IdentityConfig.store).to receive(:in_person_usps_outage_message_enabled).
+    let(:formatted_date) { 'Saturday, November 1' }
+
+    it 'renders the outage alert when flag is enabled' do
+      allow(IdentityConfig.store).to receive(:in_person_outage_message_enabled).
         and_return(true)
+      allow_any_instance_of(Idv::InPerson::ReadyToVerifyPresenter).to receive(:date).
+        and_return(formatted_date)
```

Comment:
Similar to my previous comment, I think we should mock the date config values instead of mocking a method on the presenter.